Fix error in building with USE=webkit

In file included from /var/tmp/portage/net-im/telegram-desktop-5.12.3-r2/work/tdesktop-5.12.3-full/Telegram/lib_webview/webview/webview_dialog.cpp:13:
/var/tmp/portage/net-im/telegram-desktop-5.12.3-r2/work/tdesktop-5.12.3-full/Telegram/lib_ui/ui/widgets/buttons.h:276:3: error: unknown type name 'nullptr_t'; did you mean 'std::nullptr_t'?
  276 |                 nullptr_t,
      |                 ^~~~~~~~~
      |                 std::nullptr_t
/usr/lib/gcc/x86_64-pc-linux-gnu/14/include/g++-v14/x86_64-pc-linux-gnu/bits/c++config.h:314:29: note: 'std::nullptr_t' declared here
  314 |   typedef decltype(nullptr)     nullptr_t;
      |                                 ^
1 error generated.

--- tdesktop-5.12.3-full.orig/Telegram/lib_ui/ui/widgets/buttons.cpp
+++ tdesktop-5.12.3-full/Telegram/lib_ui/ui/widgets/buttons.cpp
@@ -795,7 +795,7 @@
 
 SettingsButton::SettingsButton(
 	QWidget *parent,
-	nullptr_t,
+	std::nullptr_t,
 	const style::SettingsButton &st)
 : RippleButton(parent, st.ripple)
 , _st(st)
--- tdesktop-5.12.3-full.orig/Telegram/lib_ui/ui/widgets/buttons.h
+++ tdesktop-5.12.3-full/Telegram/lib_ui/ui/widgets/buttons.h
@@ -273,7 +273,7 @@
 		const style::SettingsButton &st = st::defaultSettingsButton);
 	SettingsButton(
 		QWidget *parent,
-		nullptr_t,
+		std::nullptr_t,
 		const style::SettingsButton &st = st::defaultSettingsButton);
 	~SettingsButton();
 
